local/config/datavault.properties

```diff
@@ -0,0 +1,166 @@
+# 1MB chunks
+chunking.size = 1MB
+# encrypt tar before archiving
+encryption.enabled = true
+encryption.mode = GCM
+# 32K buffer to match CPU L1 cache
+encryption.bufferSizeInByte = 30KiB
+
+max.deposit.size = 1GiB
+
+# Email server settings
+# =====================
+# The email account of the system administrator
+mail.administrator = feedback@datavaultplatform.org
+# SMTP host for sending mail
+mail.host = smtp.gmail.com
+# SMTP port for sending mail
+mail.port = 587
+# SMTP account name for sending mail
+mail.username = XXX@gmail.com
+# SMTP password for sending mail
+mail.password = XXX
+
+# Welcome settings
+# ================
+# Welcome message (HTML) displayed by the (non-shibboleth) login page.
+webapp.welcome =
+# Message of the day - Used for alerts and such like
+webapp.motd =
+
+
+# CRIS system settings
+# ====================
+# The URL of the external metadata service (for example, a Pure CRIS API). Leave this blank to use a mock provider
+# eg. https://example.org/ws/rest/datasets
+# If using HTTP BASIC authentication, use: https://username:password@example.org/ws/rest/datasets
+metadata.url =
+# Name displayed in the help page for the institutional CRIS or external metadata system
+metadata.system = CRIS
+# URL displayed in the help page for the institutional CRIS or external metadata system
+metadata.link = http://cris.example.com/
+
+# Retention policy checker settings
+# =================================
+# How frequently the retention policy check job is run (in 'cron' format)
+# Example: Check the retention policies each weekday at 9am
+retentioncheck.schedule = 0 0 20 * * MON-FRI
+encryptioncheck.schedule = 0 0 21 * * *
+
+# SFTP server settings
+# ====================
+# Default SFTP host
+sftp.host = localhost
+# Default SFTP port
+sftp.port = 22
+# Default SFTP path
+sftp.rootPath = /
+# Default SFTP key passphrase
+sftp.passphrase = datavault
+
+# Shibboleth authentication settings
+# ==================================
+# Shibboleth header containing the username
+shibboleth.principal= uid
+# Shibboleth header containing the user's first name
+shibboleth.firstname = givenName
+# Shibboleth header containing the user's last name
+shibboleth.lastname = sn
+# Shibboleth header containing the user's email
+shibboleth.email = mail
+
+# LDAP settings
+# =============
+# Set this to false if you don't need to retrieve attributes from LDAP
+ldap.enabled = false
+ldap.host = hostname
+ldap.port = 636
+ldap.useSsl = true
+# In this example the uun is an account with privileged access to LDAP
+ldap.dn = uid=uun,ou=people,o=myu.ed
+ldap.password = secret
+ldap.searchContext = ou=people,o=myu.ed
+ldap.searchFilter = uid
+# A list of the attributes you want to retrieve from LDAP
+ldap.attrs = attr1,attr2,etc
+
+# AWS settings
+# ===========
+s3.bucketName = datavault-archive
+s3.region = eu-west-1
+s3.awsAccessKey = -
+s3.awsSecretKey =  -
+
+# HashCorp Vault
+# ===========
+vault.enable = true
+vault.address = http://vault:8200
```

Comment:
I had to change this to `http://localhost:8200` to be able to set up the sftp connection on my local environment. I've not got it fully working yet so I wouldn't bet my life on this block not needing any more changes, but I'm pretty certain the issue is with the sftp server auth config rather than with anything here so it might be worth setting that with this set of changes